### ISSUE TITLE
Calendar form field on('change') event handler not working

### ISSUE THREAD

**Issue by skondakov** [atk4/ui#521]
When I set an on change event handler on a Calendar form field, even a simple alert, it is not being executed.

**Comment by ibelar** [comment]
@skondakov - How do you setup your onChange handler?

This is working fine for me and I can properly see output in console:

```
$f = $app->add('Form');
$f->setModel($m);

$d = $f->fields['date'];
$d->options['onChange'] = new \atk4\ui\jsExpression('function(date, text, mode){
            console.log(date, text, mode);
            }');
```

I simply pass the onChange option to the date field passing a jsExpression to it.

**Comment by romaninsh** [comment]
I think he uses jsInput()->on('change', ... ) , which works with regular inputs, but is not being triggered by semantic ui. In which case it probably is a femantic's issue.

If that's the case, perhaps Femantic-UI can do it (see also #523)

**Comment by ibelar** [comment]
Seems like fomantic is not using any calendar input yet...
https://github.com/hammy2899/Fomantic-UI/issues/8

And in our case, we are using mdehoog calendar input: https://github.com/mdehoog/Semantic-UI-Calendar

Mdehoog input use it's own 'change' event handler internally for which you can hook up using the 'onChange' option of the calendar input.

**Comment by romaninsh** [comment]
could be solved by #523

**Comment by DarkSide666** [comment]
Switched to Fomantic. Need to test if this is now ok and if it is - close ticket.

**Comment by ibelar** [comment]
Fix in #547
